client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -253,14 +255,15 @@ export const CreateEncounterModal: FC = () => {
       width={700}
     >
       <React.Suspense fallback={<div />}>
-        <Stack alignItems="flex-start" gap={1} sx={{ paddingLeft: '20px' }}>
+        <Stack gap={1} sx={{ padding: '20px' }}>
           <InputWithLabelRow
+            labelProps={{ sx: { flex: `${LABEL_FLEX}` } }}
```

Comment:
```suggestion
            labelProps={{ sx: { flex: LABEL_FLEX } }}
```

String interpolation shouldn't be needed here 😁